lib/pages/caregiver/panel_page.dart

```diff
@@ -60,14 +62,14 @@ class _CaregiverPanelPageState extends State<CaregiverPanelPage> {
 				title: '$_pageKey.content.caregiverProfilesTitle',
 				subtitle: '$_pageKey.content.caregiverProfilesSubtitle',
 				noElementsMessage: '$_pageKey.content.noCaregiverProfilesAdded',
-				headerAction: UIButton('$_pageKey.header.addCaregiver', () => {}, AppColors.caregiverButtonColor, Icons.add),
+				headerAction: UIButton('$_pageKey.header.addCaregiver', () => showAddFriendDialog(context, () => context.bloc<CaregiverPanelCubit>().doLoadData()), AppColors.caregiverButtonColor, Icons.add),
```

Comment:
What's not working without that refresh callback? We should be able to handle it in a more elegant way

Reply:
I tried to do it with user auth state update, but it didn't work. It needs to get friend name and reload the list (only the code is provided). Probably there is a way, I just don't know how to do it :c